tests/functional/adapter/iceberg/test_iceberg_support.py

```diff
@@ -5,7 +5,8 @@
 from dbt.artifacts.schemas.results import RunStatus
 
 
-@pytest.mark.skip_profile("databricks_cluster")
+# @pytest.mark.skip_profile("databricks_cluster")
```

Comment:
why do you comment out instead of deleting it?

Reply:
It's only temporary until next week, when the fix will get released to prod.